File: assets/cmp.c
int i;
int main() {
    int j;
    i = 5;
    j = 6;
    printf("Yes:\n");
    printf("\ti < j: %d\n", i < j);
    printf("\ti <= j: %d\n", i <= j);
    printf("\ti == i: %d\n", i == i);
    printf("\t!(j <= i): %d\n", !(j <= i));
    printf("\t!(j < i): %d\n", !(j < i));
    printf("\tj > i: %d\n", j > i);
    printf("\tj >= i: %d\n", j >= i);
    printf("\tj != i: %d\n", j != i);
    printf("No:\n");
    printf("\tj < i: %d\n", j < i);
    printf("\tj <= i: %d\n", j <= i);
    printf("\ti == j: %d\n", i == j);
    printf("\t!(i < j): %d\n", !(i < j));
    printf("\t!(i <= j): %d\n", !(i <= j));
    printf("\ti > j: %d\n", i > j);
    printf("\ti >= j: %d\n", i >= j);
    printf("\ti != i: %d\n", i != i);
}

File: verify/calc/cmp.py
def verify(*args):
    i = 5;
    j = 6;
    res = ""
    res += "Yes:\n"
    res += "\ti < j: {}\n".format(int(i < j))
    res += "\ti <= j: {}\n".format(int(i <= j))
    res += "\ti == i: {}\n".format(int(i == i))
    res += "\t!(j <= i): {}\n".format(int(not (j <= i)))
    res += "\t!(j < i): {}\n".format(int(not (j < i)))
    res += "\tj > i: {}\n".format(int(j > i))
    res += "\tj >= i: {}\n".format(int(j >= i))
    res += "\tj != i: {}\n".format(int(j != i))
    res += "No:\n"
    res += "\tj < i: {}\n".format(int(j < i))
    res += "\tj <= i: {}\n".format(int(j <= i))
    res += "\ti == j: {}\n".format(int(i == j))
    res += "\t!(i < j): {}\n".format(int(not (i < j)))
    res += "\t!(i <= j): {}\n".format(int(not (i <= j)))
    res += "\ti > j: {}\n".format(int(i > j))
    res += "\ti >= j: {}\n".format(int(i >= j))
    res += "\ti != i: {}\n".format(int(i != i))
    return (0, res, "")

data = [
    [],
]
